workloads/templates/common-templates.md

```diff
@@ -0,0 +1,38 @@
+# Virtual machine templates
+
+## What is a virtual machine template?
+
+The KubeVirt projects provides a set of [templates](https://docs.okd.io/latest/dev_guide/templates.html) to create VMs to handle common usage scenarios.
+These templates provide a combination of some key factors that could be further customized and processed to have a Virtual Machine object.
+The key factors which define a template are
+- Workload
+most Virtual Machine should be *generic* to have maximum flexibility; the *highperformance* workload trades some of this flexibility to
+provide better performance
+- Guest Operating System (OS)
+Enable optimizations depending on the OS which is going to run on the guest.
```

Comment:
Also makes sure the default emulated hardware is compatible with the OS.

Reply:
Thanks @fromanirh, I've add reference to your doc from my own (I've just add it to the PR)

Reply:
@fromanirh how do you think, maybe good idea to add information regarding webui? Just something very short - that it's exist and support templates?
You can check my paragraph: https://github.com/sHaggYcaT/user-guide/blob/8epic-templating-14-user-guide-section-for-the-use-of-templates-initiative-document-happy-path/workloads/templates/vm-creation.md
I not sure is it information enough, or otherwise, more then enough.

Reply:
@sHaggYcaT I think what you added is just fine. It mentions the existing web-ui, so users can look up for it if they want, in the web ui official docs.

Reply:
> Also makes sure the default emulated hardware is compatible with the OS.

done (and rephrased)